nautobot/dcim/tests/test_views.py

```diff
@@ -4592,9 +4591,14 @@ def test_correct_handling_for_model_protected_error(self):
             "confirm": True,
             "_confirm": True,  # Form button
         }
-        response = self.client.post(self._get_url("bulk_delete"), data, follow=True)
-        # Assert protected error message included in the response body
-        self.assertBodyContains(response, f"<span>{device_type_to_software_image_file}</span>", html=True)
+        response = self.client.post(self._get_url("bulk_delete"), data)
+        job_result = JobResult.objects.filter(name="Bulk Delete Objects").first()
+        self.assertRedirects(
+            response,
+            reverse("extras:jobresult", args=[job_result.pk]),
+            status_code=302,
+            target_status_code=200,
+        )
```

Comment:
This change eliminates the purpose of this test case, which is to make sure that the bulk-delete fails due to the ProtectedError and reports things clearly to the user. Instead of rewriting it like so, I'd expect it to be removed from this file and reimplemented as an additional test case in `test_jobs.py` above.